packages/components/src/__actions__/Button/v3/subcomponents/PendingContent/PendingContent.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import { LoadingSpinner } from "~components/Loading"
+import { VisuallyHidden } from "~components/VisuallyHidden"
+import { mergeClassNames } from "~components/utils/mergeClassNames"
+import { ButtonSizes, PendingProps } from "../../types"
+import styles from "./PendingContent.module.css"
+
+export const PendingContent = ({
+  pendingLabel,
+  isPendingLabelHidden = false,
+  size = "medium",
+}: PendingProps & { size?: ButtonSizes }): JSX.Element => (
+  <span className={mergeClassNames(styles.pendingContent, styles[size])}>
```

Comment:
`mergeClassNames` only makes sense to use for RAC styling
```suggestion
  <span className={classNames(styles.pendingContent, styles[size])}>
```

Reply:
That's fair, probs best to keep it isolated to the top level of RAC